course_discovery/apps/api/tests/test_utils.py

```diff
@@ -40,3 +44,14 @@ def test_with_request(self):
 
     def test_without_request(self):
         assert get_query_param(None, 'q') is None
+
+
+class TestGetQuerysetFilteredOnOrganization:
+
+    def test_courses_filter_on_edx_shortname(self):
+        queryset = mock.MagicMock()
```

Comment:
This test does not make sense. Can we use actual queryset and filter it instead of mock queryset?

Reply:
@ziafazal Updated test.